### DISCUSSION TITLE
CNCF TAG-Runtime Discussion/Presentation (?)

### DISCUSSION THREAD

**Discussion by raravena80** [slimtoolkit/slim#487] [General]
Hello slim team,

I'm one of the co-chairs of the CNCF TAG-Runtime. I'm reaching out because we'd love for you to present/discuss the slim project in one of our meetings.  Let us know if that's something you'd like to do.

Feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com) at your convenience. 

Thanks!

**Comment by kcq** [comment]
@raravena80 absolutely. There's a lot of interesting things that can be done already with Slim and there's a number of new capabilities planned that will make it even more useful at runtime in the containerized environments.
